nornir_ansible/plugins/inventory/ansible.py

```diff
@@ -178,10 +181,17 @@ def _vars_file_exists(path: str) -> bool:
             bool: True if a file exists with of of the extension
 
         """
-        for ext in VARS_FILENAME_EXTENSIONS:
-            if Path(f"{path}{ext}").is_file():
-                return True
-        return False
+        if not Path(f"{path}{host}").is_dir():
+          exts = '|'.join(map(str,VARS_FILENAME_EXTENSIONS))
+          pattern = re.compile(r'.*{}({})$'.format(re.escape(host), exts), re.IGNORECASE)
+          found_files = [f for f in os.listdir(path) if pattern.search(f)]
```

Comment:
could we just glob here w/ path lib so we can let them deal w/ building the list? (im not actually sure offhand, just asking the question!)